recipes/aws-libfabric/all/conanfile.py

```diff
@@ -0,0 +1,113 @@
+from conans import ConanFile, AutoToolsBuildEnvironment, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+required_conan_version = ">=1.35.0"
+
+class LibfabricConan(ConanFile):
+    name = "aws-libfabric"
+    description = "AWS Libfabric"
+    topics = ("fabric", "communication", "framework", "service")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/aws/libfabric"
+    license = "BSD-2-Clause", "GPL-2.0-or-later"
+    settings = "os", "arch", "compiler", "build_type"
+    _providers = ['gni', 'psm', 'psm2', 'sockets', 'rxm', 'tcp', 'udp', 'usnic', 'verbs', 'bgq', 'shm', 'efa', 'rxd', 'mrail', 'rstream', 'perf', 'hook_debug']
+    options = {
+        **{ p: "ANY" for p in _providers },
+        **{
+            "shared": [True, False],
+            "fPIC": [True, False],
+            "with_libnl": [True, False],
+            "with_bgq_progress": [None, "auto", "manual"],
+            "with_bgq_mr": [None, "basic", "scalable"]
```

Comment:
I think `False` makes more sense here.
```suggestion
            "with_bgq_progress": [False, "auto", "manual"],
            "with_bgq_mr": [False, "basic", "scalable"]
```